ui/src/pages/backoffice/Document/DocumentDetails/components/DocumentRelations/components/RelationListEntry/RelationListEntry.js

```diff
@@ -0,0 +1,73 @@
+import { recordToPidType } from '@api/utils';
+import { DocumentAuthors, SeriesAuthors } from '@components';
+import {
+  DocumentEdition,
+  DocumentItemCover,
+  DocumentTitle,
+} from '@components/Document';
+import { BackOfficeRoutes } from '@routes/urls';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Grid, Icon, Item } from 'semantic-ui-react';
+
+export class RelationListEntry extends Component {
```

Comment:
Since everything is called item, like `<Item`, "item-image-wrapper" etc should we call this `RelationListItem` ? I see where this can actually become a problem when working with `items`, `eitems` though.

Reply:
there is also `<Item>` as semantic-ui component, I didn't want to mix the names